scripts/pi/wpa_supplicant.conf

```diff
@@ -3,7 +3,7 @@ ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
 update_config=1
 
 network={
-    # bssid="YOUR_2_4_GHZ_BSSID"
+    #bssid=YO:UR:24:GH:ZB:SS:ID
```

Comment:
this actually got me thinking there is one thing I will need to fix (as mine didn't have the comment as I changed to include bssid).

- [ ] support removing the comment if one exists in the sed command